main.go

```diff
@@ -100,6 +110,12 @@ func main() {
 			Usage:  "Private key used in client TLS auth",
 			Value:  "",
 		},
+		cli.StringFlag{
+			EnvVar: "MACHINE_SSH_CLIENT_TYPE",
+			Name:   "ssh-client-type",
+			Usage:  "Type of SSH client to use.  Options: gonative, shellout",
+			Value:  "",
```

Comment:
Yeah, I like that- I guess I had some kind of hazy idea that maybe at some point other SSH backends might be implemented, but that seems kind of like a YAGNI thing to me at this point since I can't think of any obvious examples.